pkg/controller/usbdevice/usbdevice_controller.go

```diff
@@ -174,14 +180,32 @@ func (h *DevHandler) handleList(createList []*v1beta1.USBDevice, updateList []*v
 		}
 	}
 
-	for _, usbDevice := range updateList {
+	for _, usbDevice := range usageList.updateList {
 		if _, err := h.usbClient.UpdateStatus(usbDevice); err != nil {
 			logrus.Errorf("failed to update existed USB device status: %v\n", err)
 			return err
 		}
 	}
 
-	for _, usbDevice := range deleteList {
+	for _, usbDevice := range usageList.orphanedList {
+		err := h.usbClaimClient.Delete(usbDevice.Name, &metav1.DeleteOptions{})
+
+		if err == nil {
+			continue
+		}
```

Comment:
Do we need to verify if the error is from webhook?

Reply:
I don't think we need to do that. Regardless of any errors, we just need to mark it as orphaned since the USB device is already not present on the host.